kernelci.org/content/en/kernel-community/view-results.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "Evaluating Test Results"
+date: 2024-09-06
+description: "Describe ways to interact with the KernelCI test results"
+---
+
+The Kernel Community have a few options to interact with the test results available in the KernelCI. Remember this data include results from several CI systems.
+
+* [**Web Dashboard**](https://dashboard.kernelci.org/)(WIP): Our new Web Dashboard aims at providing an easy way for the community to look at the tests results. It is still under development and we open to your [feedback and feature requests](https://github.com/kernelci/dashboard).
+* [**Grafana**](https://grafana.kernelci.org/): Our Grafana instance allow users to create specific dashboard with tailored queries and boards. Those who want to dive deeper in their data can engage with our Grafana instance.
+* **Email reports**: [KCIDB](../../kcidb) allows users to create tailored email reports through [templates](https://github.com/kernelci/kcidb/tree/main/kcidb/templates).
```

Comment:
Do you think it is worth mentioning the mailing list (staging results) we are sending email reports to?

Reply:
Yes. Just added a mention to that.